record_web/lib/record_web.dart

```diff
@@ -231,8 +270,15 @@ class RecordPluginWeb extends RecordPlatform {
 
       _mediaStream = null;
     }
-
     _chunks = [];
+    try {
+      if (audioCtx?.state != 'closed') {
```

Comment:
if `audioCtx` is null the condition will be true

Reply:
done